std/math.d

```diff
@@ -133,8 +133,15 @@ static import core.stdc.fenv;
 import std.traits :  CommonType, isFloatingPoint, isIntegral, isNumeric,
     isSigned, isUnsigned, Largest, Unqual;
 
+// @@@DEPRECATED_2.102@@@
 // Note: Exposed accidentally, should be deprecated / removed
-public import std.meta : AliasSeq;
+private template DeprecatedAliasSeq(V...)
```

Comment:
I think you need to make this public, no?

Reply:
No, otherwise we'd have to deprecated the DeprecatedAliasSeq too.

Reply:
Can you publicly alias a private symbol? I didn't think that was possible.

Reply:
It works for me.

Reply:
OK, you are right, I tested locally and it works.